shared/README.md

```diff
@@ -175,7 +180,7 @@ set `KEYBASE_DEV_TOOL_EXTENSIONS` instead of `KEYBASE_DEV_TOOL_ROOTS`,
 and you'll have to use the version subdirectory:
 
 ```
-set -x KEYBASE_DEV_TOOL_EXTENSIONS "$HOME/Library/Application Support/Google/Chrome/Default/Extensions/fmkadmapgofadopljbjfkapdkoienihi/2.5.0_0"
+set -x KEYBASE_DEV_TOOL_EXTENSIONS "$HOME/Library/Application Support/Google/Chrome/Default/Extensions/fmkadmapgofadopljbjfkapdkoienihi/2.5.2_0,$HOME/Library/Application Support/Google/Chrome/Default/Extensions/hgldghadipiblonfkkicmgcbbijnpeog/1.7_0"
 ```
```

Comment:
i'd add a a note that those version numbers change automatically when chrome updates them so they might break at any time

Reply:
There's already a note below, but updated it to say that it can happen at any time.

Reply:
I thought using `KEYBASE_DEV_TOOL_ROOTS` is more stable for this because it omits the version numbers